.github/workflows/build-and-push-ui-images.yml

```diff
@@ -0,0 +1,94 @@
+name: Build and Push UI and BFF Images
+on:
+  pull_request:
```

Comment:
I appreciate in the PR description is mentioned this `on: pull_request` is only for testing, but I'd rather make a comment so to make sure we don't forget before merging 😅